### PR TITLE
Clarify DateBox - disabledDates rules for different views

### PR DESCRIPTION
**Pull request by czernous-devexpress**

### CODE REVIEW

File: api-reference/10 UI Widgets/dxDateBox/1 Configuration/disabledDates.md [hunk 1]
```diff
@@ -142,7 +142,8 @@ The current view: *"month"*, *"year"*, *"decade"*, or *"century"*.
 
 ---
 
-... or a function that specifies whether the currently checked date is disabled.
+... or a function that defines rules controlling whether the currently checked date is disabled.
+Make sure to verify that these rules are applied to the desired view.
```
Comment (RomanTsukanov) [inline api-reference/10 UI Widgets/dxDateBox/1 Configuration/disabledDates.md]: These two lines will be displayed as a single line. Let's make it a single line in the document too.

**Review by RomanTsukanov — APPROVED** [review]

File: api-reference/10 UI Widgets/dxDateBox/1 Configuration/disabledDates.md [hunk 2]
```diff
@@ -142,7 +142,7 @@ The current view: *"month"*, *"year"*, *"decade"*, or *"century"*.
 
 ---
 
-... or a function that specifies whether the currently checked date is disabled.
+... or a function that defines rules controlling whether the currently checked date is disabled. Make sure to verify that these rules are applied to the desired view.
```
Comment (dirk-pieterse) [inline api-reference/10 UI Widgets/dxDateBox/1 Configuration/disabledDates.md]:
```suggestion
... or a function that defines the rules that specify whether the checked date is disabled. You should verify that these rules are applied to a specific view.
```

Comment (dirk-pieterse) [reply api-reference/10 UI Widgets/dxDateBox/1 Configuration/disabledDates.md]: "Make sure to verify that these rules are applied to the desired view" I'm not entirely sure what this purpose is of this statement. What does 'desired' refer to?

**Review by dirk-pieterse — APPROVED** [review]

File: api-reference/10 UI Widgets/dxDateBox/1 Configuration/disabledDates.md (hunk 2 above)
Comment (czernous-devexpress) [reply api-reference/10 UI Widgets/dxDateBox/1 Configuration/disabledDates.md]: The purpose of this statement is to indicate that every view should be individually targeted by a separate set of rules. As far as 'desired' goes, it refers to a view that the user intends to target by the rules. Perhaps 'targeted view' as a better alternative.
'Specify' is probably not the best word, as it was used earlier in the text https://js.devexpress.com/Documentation/ApiReference/UI_Widgets/dxDateBox/Configuration/#disabledDates

"Specifies dates that users cannot select." and "... or a function that specifies..."

Could you also please clarify rephrasing "Make sure to..." as "You should..."?

Comment (dirk-pieterse) [reply api-reference/10 UI Widgets/dxDateBox/1 Configuration/disabledDates.md]: - You can use 'specify' more than once in a doc - the point is not to overuse it, or use it in the same or subsequent sentences thereby creating a pattern
-  'Make sure to' is an instruction for the reader, you can rephrase this as 'You should do something' since you are telling the reader to do something
- 'Make sure to verify that these rules are applied to the desired view' and 'every view should be individually targeted by a separate set of rules' are not the same thing. You should consider rephrasing this sentence

File: api-reference/10 UI Widgets/dxDateBox/1 Configuration/disabledDates.md [hunk 3]
```diff
@@ -142,7 +142,7 @@ The current view: *"month"*, *"year"*, *"decade"*, or *"century"*.
 
 ---
 
-... or a function that specifies whether the currently checked date is disabled.
+Alternatively, pass a function to disabledDates. This function should define the rules that determine whether the checked date is disabled. A separate set of rules should target every [view](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/views/) individually.
```
Comment (RomanTsukanov) [inline api-reference/10 UI Widgets/dxDateBox/1 Configuration/disabledDates.md]:
```suggestion
Alternatively, pass a function to **disabledDates**. This function should define the rules that determine whether the checked date is disabled. A separate set of rules should target every [view](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/views/) individually.
```

**Review by RomanTsukanov — APPROVED** [review]
